osgp/protocol-adapter-dlms/osgp-protocol-adapter-dlms/src/test/java/org/opensmartgridplatform/adapter/protocol/dlms/domain/commands/firmware/UpdateFirmwareCommandExecutorIntegrationTest.java

```diff
@@ -50,94 +50,178 @@ class UpdateFirmwareCommandExecutorIntegrationTest {
   @Mock private FirmwareFileCachingRepository firmwareFileCachingRepository;
   @Mock private FirmwareImageIdentifierCachingRepository firmwareImageIdentifierCachingRepository;
   @Mock private MacGenerationService macGenerationService;
-  @Mock private UpdateFirmwareConfig updateFirmwareConfig;
 
   private DlmsConnectionManagerStub connectionManagerStub;
   private DlmsConnectionStub connectionStub;
   private MessageMetadata messageMetadata;
 
-  private final int verificationStatusCheckInterval = 1;
-  private final int verificationStatusCheckTimeout = 2;
-  private final int initiationStatusCheckInterval = 3;
-  private final int initiationStatusCheckTimeout = 4;
+  final byte[] firmwareFileNotMbus =
+      org.bouncycastle.util.encoders.Hex.decode(
+          "0000000000230011004000310000001000020801e91effffffff500303000000000000831c9d5aa5b4f"
+              + "fbfd057035a8a7896a4abe7afa36687fbc48944bcee0343eed3a75aab882ec1cf57820adfd4394e262d"
+              + "5fa821c678e71c05c47e1c69c4bfffe1fd");
+  final String fwFileHash = "951948459d2b7b59883cfc75c2bc2b7e4a0232ae7973a0d99526afd9458b0c86";
+  final byte[] firmwareFileMbus =
+      org.bouncycastle.util.encoders.Hex.decode(
+          "534d523500230011004000310000001000020801e91effffffff500303000000000000831c9d5aa5b4f"
+              + "fbfd057035a8a7896a4abe7afa36687fbc48944bcee0343eed3a75aab882ec1cf57820adfd4394e262d"
+              + "5fa821c678e71c05c47e1c69c4bfffe1fd");
 
   @BeforeEach
   void setUp() {
+
+    final int verificationStatusCheckInterval = 1;
+    final int verificationStatusCheckTimeout = 2;
+    final int initiationStatusCheckInterval = 3;
+    final int initiationStatusCheckTimeout = 4;
```

Comment:
Why are these variables defined? The values are only used once and the variable name are not more descriptive than the method names (`verificationStatusCheckInterval` is only used for `imageTransferProperties.setVerificationStatusCheckInterval()`)

Reply:
variables removed.